pkg/dataplane/internal/challenge/Challenge.g4

```diff
@@ -0,0 +1,240 @@
+// $antlr-format alignTrailingComments true, columnLimit 150, minEmptyLines 1, maxEmptyLinesToKeep 1, reflowComments false, useTab false
```

Comment:
is this file hand crafted or copied from somewhere else?

Reply:
This `.g4` specification file is hand-crafted, the `challenge_*.go` are generated - please see the Makefile.